### DISCUSSION TITLE
Tree.onNodeDrop: Differentiate between Reordering and Nesting

### DISCUSSION THREAD

**Discussion by S-Mitterlehner** [primefaces/community#1091] [PrimeNG]
When using the tree component (with drag and drop) to reorder items, it's not possible to determine whether it's sorting (dropping on droppoint) or nesting (dropping on element).
At the moment, I have to check the css class like this to differeniate: `(event.originalEvent.target as HTMLElement)?.classList.contains('p-treenode-droppoint')`. 

Maybe there is a better approach I missed, but if not my proposal is to add a `destinationType` to the TreeDropEvent:
```ts
export enum DestinationType {
  Node,
  Droppoint
}
```

**Comment by cetincakiroglu** [comment]
Hi,

To better understand the case, could you please share a stackblitz example?

**Comment by cetincakiroglu** [comment]
Closing the discussion due to inactivity
